vignettes/documentation_singleRData.Rmd

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Is this document necessary? Just move the details into the function documentation, with a brief mention about the available datasets in the main vignette (if it isn't there already).

Reply:
I'd like to keep this for now as a means to collect whatever tidbits of information I can get; it's still very much a draft, but eventaully I'd like to include more details about the actual cell types present in the individual data sets so that users can judge whether their cell types of interest are better represented by one reference data set over another.